locales/config.yml

```
@@ -5,6 +5,8 @@ db_default_goodbye: "Sad to see you leaving {first}"
 db_warn_no_reason: "No Reason"
 alt_names:
   Admin: [admins, promote, demote, title]
+  Approvals: [approval, approve, unapprove]
+  Approvals: [approval, approve, unapprove]
```

YAMLlint (1.38.0)

[error] duplication of key "Approvals" in mapping

(key-duplicates)